pkg/qemu/qemu.go

```diff
@@ -43,7 +43,10 @@ type Config struct {
 }
 
 // MinimumQemuVersion is the minimum supported QEMU version
-const MinimumQemuVersion = "4.0.0"
+const (
+	MinimumQemuVersion = "4.0.0"
+	LimaGuestSock      = "lima-guest.sock"
```

Comment:
Can this be called `ga.sock` to match `ha.sock` ?

Reply:
This was un-used, removed it